packaging/nix/default.nix

```diff
@@ -0,0 +1,42 @@
+{ pkgs ? import <nixpkgs> {} }:
+# Based on nixpkgs/pkgs/applications/graphics/darktable/default.nix
+
+let stdenv = pkgs.stdenv;
+in stdenv.mkDerivation rec {
+  name = "darktable-git";
+
+  src = pkgs.lib.cleanSource ./.;
+
+  buildInputs = with pkgs; with xorg; with gnome2;
```

Comment:
I took these from the [Nixpkgs derivation](https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/graphics/darktable/default.nix), which I didn't write in the first place. However, it seems you're right, as I was able to remove all the ones you said and the build appeared fine to me.